api/swagger.yml

```diff
@@ -2383,6 +2389,14 @@ paths:
           required: false
           schema:
             type: string
+        - in: header
+          name: If-None-Match
+          description: Currently supports only "*" to allow uploading an object only if one doesn't exist yet.
+          example: "*"
+          required: false
+          schema:
+            type: string
+            pattern: '\*'  # Currently, only "*" is supported
```

Comment:
I think this pattern will match anything with * - we would like to limit the pattern to '^*$'